proxy.py

```diff
@@ -2,225 +2,145 @@
 import gevent.monkey; gevent.monkey.patch_all(subprocess=True)
 from flask import Flask, request, jsonify
 from flask.ext.cors import CORS, cross_origin
-import uuid
-import tempfile
-import os
-import os.path
-import errno
 import atexit
 import gevent
 import collections
-import traceback
-
+import ycm_helpers
```

Comment:
Please include internal includes after library includes (this also goes for `settings`).